File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App_Start/Startup.vb
Imports Microsoft.AspNet.SignalR
Imports Microsoft.Owin
Imports Microsoft.Owin.Cors
Imports Owin

<Assembly: OwinStartup(GetType(VecompSoftware.DocSuiteWeb.Gui.Startup))>
Public Class Startup
    Public Sub Configuration(app As IAppBuilder)
        app.Map("/signalr", Sub(map)
                                map.UseCors(CorsOptions.AllowAll)
                                Dim hubConfiguration As HubConfiguration = New HubConfiguration() With {.EnableJSONP = True}
#If DEBUG Then
                                hubConfiguration.EnableDetailedErrors = True
#End If
                                map.RunSignalR(hubConfiguration)
                            End Sub)
    End Sub
End Class

File: SOURCE_BIBLOSDS/VecompSoftware.BiblosDS.WindowsService.WebAPI/Startup.cs
using Microsoft.Owin;
using Owin;

[assembly: OwinStartup(typeof(VecompSoftware.BiblosDS.WindowsService.WebAPI.Startup))]

namespace VecompSoftware.BiblosDS.WindowsService.WebAPI
{
    public partial class Startup
    {
        public void Configuration(IAppBuilder app)
        {
            ConfigureAPI(app);
        }
    }
}
